catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsFilterDelegate.java

```diff
@@ -559,22 +542,18 @@ private FilterType buildPropertyIsFilterForMultipleProperties(
     featureMetacardType
         .getTextualProperties()
         .forEach(
-            new Consumer<String>() {
```

Comment:
So this change here is functionally equivalent?  We just wanted to go from a consumer to a lambda.  Am I seeing this right?

Reply:
correct, intellij usually flags these cases with a warning